rotkehlchen/chain/ethereum/modules/aave/graph.py

```diff
@@ -565,6 +639,53 @@ def _process_events(
             total_earned_liquidations=total_earned_liquidations,
         )
 
+    def _process_graph_query_result(
+        self,
+        query: Dict[str, Any],
+        deposits: List[AaveDepositWithdrawalEvent],
+        withdrawals: List[AaveDepositWithdrawalEvent],
+        borrows: List[AaveBorrowEvent],
+        repays: List[AaveRepayEvent],
+        liquidation_calls: List[AaveLiquidationEvent],
+        user_merged_data: Dict[str, Any],
+        from_ts: Timestamp,
+        to_ts: Timestamp,
+    ) -> None:
+        """
+        Given a query result from the graph this function extracts information for:
+        - deposits
+        - withdrawals
+        - borrows
+        - repays
+        - liquidation_calls
+        and extends the corresponding arguments with the obtained information.
+        """
+        if 'users' not in query or len(query['users']) == 0:
+            # If there is no information on the query finish the execution
+            log.debug(f'Aave subgraph query has no information for user. {str(query)}')
+            return
+        user_result = query['users'][0]
```

Comment:
nitpick: I say all the time you are putting a lot of unnecessary whitespace. But here, you can be. It separates a section. Makes the return a lot more visible.
```suggestion
            return
        user_result = query['users'][0]
```